src/near_indexer/src/helper/twitter.rs

```diff
@@ -0,0 +1,95 @@
+use reqwest;
+use reqwest_oauth1::OAuthClientProvider;
+use serde_json::json;
+use serde_json::Value;
+use std::env;
+use std::error::Error;
+use std::marker::{Send, Sync};
+
+pub struct OathTweeterHandler {
+    consumer_key: String,
+    consumer_secret: String,
+    access_token: String,
+    token_secret: String,
+}
+
+impl OathTweeterHandler {
+    /// Creates a default `OathTweeterHandler`.
+    pub fn default() -> Self {
+        let consumer_key = env::var("TWEET_CONSUMER_KEY").expect("TWEET_CONSUMER_KEY_NOT_PRESENT");
+        let consumer_secret =
+            env::var("TWEET_CONSUMER_SECRET").expect("TWEET_CONSUMER_SECRET_NOT_PRESENT");
+        let access_token = env::var("TWEET_ACCESS_TOKEN").expect("TWEET_ACCESS_TOKEN_NOT_PRESENT");
+        let token_secret = env::var("TWEET_TOKEN_SECRET").expect("TWEET_TOKEN_SECRET_NOT_PRESENT");
+        return Self::new(
+            &consumer_key,
+            &consumer_secret,
+            &access_token,
+            &token_secret,
+        )
+        .unwrap();
+    }
+
+    /// Creates a new `OathTweeterHandler`.
+    pub fn new(
+        consumer_key: &str,
+        consumer_secret: &str,
+        access_token: &str,
+        token_secret: &str,
+    ) -> Result<Self, Box<dyn Error + Send + Sync>> {
+        Ok(OathTweeterHandler {
+            consumer_key: consumer_key.to_string(),
+            consumer_secret: consumer_secret.to_string(),
+            access_token: access_token.to_string(),
+            token_secret: token_secret.to_string(),
+        })
+    }
+
+    pub async fn notifier(
```

Comment:
So we're not submitting notifications as comments to an original tweet anymore?
All notifications are now independent tweets, correct?

Reply:
Yeah, its the same cost `https://developer.x.com/en/portal/products`